### PR TITLE
use elevation-*dp variables

### PR DESCRIPTION
**Pull request by morethanreal**
@cdata @notwaldorf PTAL

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  <script src="../webcomponentsjs/webcomponents-lite.js"></script>
+
+  <link rel="import" href="../polymer/polymer.html">
+  <link rel="import" href="../iron-doc-viewer/iron-doc-viewer.html">
+
+</head>
+<body>
+
+  <iron-doc-viewer></iron-doc-viewer>
```
Comment (notwaldorf) [inline index.html]: `iron-component-page`?

File: paper-card.html [hunk 2]
```diff
@@ -33,23 +33,23 @@
...
     }
 
     :host([elevation="4"]) {
-      mixin(--shadow-elevation-4);
+      mixin(--shadow-elevation-8dp);
     }
 
     :host([elevation="5"]) {
-      mixin(--shadow-elevation-5);
+      mixin(--shadow-elevation-16dp);
```
Comment (notwaldorf) [inline paper-card.html]: `<i-dont-know-what-im-doing-dog>` I don't know anything about `dp`, but the other ones were *2. Should this be `10dp`? `</i-dont-know-what-im-doing-dog>`

**Comment by notwaldorf** [comment]
LGTM % comments and a question

File: paper-card.html (hunk 2 above)
Comment (morethanreal) [inline paper-card.html]: There's not really a mapping from `elevation` values to `dp` values. I only had hand-tuned shadows with `dp` values for 2,3,4,6,8 and 16.

File: index.html (hunk 1 above)
Comment (morethanreal) [inline index.html]: @garlicnation says don't change for now.

**Pull request merged**